aws/components/lb/setup.ftl

```diff
@@ -299,6 +299,8 @@
 
                 [#if fqdn?has_content]
                     [#local listenerRuleConditions += getListenerRuleHostCondition(fqdn) ]
+                [#else]
+                    [@fatal message="Missing FQDN" context=solution /]
```

Comment:
I think this needs to be more descriptive since we don't have a one to one map between the FQDN and a solution configuration options.

Reply:
We also don't want the error to hide the template content so the fatal should be in its own if statement so that we can still see the template content when debugging

Reply:
For the first comment, I thought the context would carry that information, have I missed something (the message was just what was wrong)

For the second, cool will do

Reply:
If that error comes up can you figure out how to set the FQDN?

Reply:
No, but I was assuming that was my ignorance as it comes from `resources["listenerRule"].FQDN` where resources is the same subOccurrence as the context in the error message